docs/maintainers/dumps.rst

```diff
@@ -0,0 +1,51 @@
+Data Dumps
+==========
+
+Check FTP Dumps age script
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Dumps may fail in production due to many reasons. We have a script to check the latest dump available on the FTP is
+younger than a specified timeframe. If the latest dump is older, an email is sent to the maintainers. This email
+is usually responsible for bringing dump failures to the notice of maintainers. This script is part of the ListenBrainz
+cronjobs and is scheduled to run a few hours after the regular dump times. If dumps are not working but no email was
+received by the maintainers, it is possible that the cron jobs are not setup properly.
+
+Logs
+^^^^
+Looking at the logs is a good starting point to debug dump failures, the log file is located at :file:`/logs/dumps.log`
+inside the listenbrainz-cron-prod container at the time of writing. You can check the crontab file for the current
+location of the log file. You can open a bash shell in the container by doing something like
+:code:`docker exec -it listenbrainz-cron-prod bash`.
+
+After that you can use :command:`tail` to view the latest logs. :command:`cat` is also available but the logs file will
+be huge so the experience using it will not be nice. For example: :code:`tail -n 500 /logs/dumps.log` will list the last
+500 lines of the log file.
```

Comment:
```suggestion
This file is large, so use :command:`tail` instead of :command:`cat` to view the logs. For example: 
:code:`tail -n 500 /logs/dumps.log` will list the last 500 lines of the log file.
```